src/webpack/tagCommonJSExports.js

```diff
@@ -2,7 +2,7 @@
 
 ;(function register() { // eslint-disable-line no-extra-semi
   /* react-hot-loader/webpack */
-  if (process.env.NODE_ENV !== 'production') {
+  if (module.hot) {
```

Comment:
Similarly, `module.hot` will not be true inside Webpack loader itself, only in app's code.